test/Microsoft.ML.Tests/TrainerEstimators/TreeEstimators.cs

```diff
@@ -115,7 +115,8 @@ public void FastTreeRankerEstimator()
                 new FastTreeRankingTrainer.Options
                 {
                     FeatureColumn = "NumericFeatures",
-                    NumTrees = 10
+                    NumTrees = 10,
+                    GroupIdColumn = "Group"
```

Comment:
>GroupIdColumn [](start = 20, length = 13)

I feel like this should be "GroupColumn" to fall more inline with FeatureColumn and LabelColumn.